lib/src/components/stepper/Stepper.test.tsx

```diff
@@ -179,4 +179,119 @@ describe('Stepper', () => {
     expect(props.onStepChange).not.toHaveBeenCalled()
     expect(onClickFn).toHaveBeenCalled()
   })
+
+  it('renders in horizontal orientation', () => {
+    render(
+      <Stepper allowSkip {...props} orientation="horizontal">
+        <Stepper.StepBack label={() => 'Back'} />
+        <Stepper.Steps />
+        <Stepper.StepForward
+          label={(activeStep) => (activeStep === 2 ? 'Start' : 'Next')}
+        />
+      </Stepper>
+    )
+    expect(screen.getByTestId('horizontal')).toBeVisible
+  })
+
+  it('renders in vertical orientation', () => {
+    render(
+      <Stepper allowSkip {...props} orientation="vertical">
+        <Stepper.StepBack label={() => 'Back'} />
+        <Stepper.Steps />
+        <Stepper.StepForward
+          label={(activeStep) => (activeStep === 2 ? 'Start' : 'Next')}
+        />
+      </Stepper>
+    )
+
+    expect(screen.getByTestId('vertical')).toBeVisible
+  })
+
+  it('renders in horizontal orientation', () => {
+    render(
+      <Stepper allowSkip {...props} orientation="horizontal">
+        <Stepper.StepBack label={() => 'Back'} />
+        <Stepper.Steps />
+        <Stepper.StepForward
+          label={(activeStep) => (activeStep === 2 ? 'Start' : 'Next')}
+        />
+      </Stepper>
+    )
+
+    expect(screen.getByTestId('horizontal')).toBeVisible
+  })
+
+  it('renders the correct number of bullets when a controlled component', () => {
```

Comment:
```suggestion
  it('renders the correct number of bullets when used as a controlled component', () => {
```

Just a suggestion as it makes more sense for me?

Reply:
Actually makes more sense with your suggestion 👍